validation/genesis/genesis-allocs_test.go

```diff
@@ -221,37 +223,12 @@ func trimWeth9BytecodeMetadataHash(expectedData []byte, gotData []byte) ([]byte,
 		log.Fatalf("Key %s not found in regenerated bytecode", key)
 	}
 
-	// repeat for gotData
-	err = json.Unmarshal(gotData, &weth9GotData)
-	if err != nil {
-		fmt.Printf("error parsing registry alloc: %s", err)
-	}
-
-	if entry, ok := weth9GotData[key].(map[string]interface{}); ok {
-		if code, ok := entry["code"].(string); ok {
-			// Trim the last 100 characters (50 bytes) which is the CBOR length in octets
-			if len(code) > 100 {
-				entry["code"] = code[:len(code)-100]
-			} else {
-				fmt.Println("The registry bytecode length is less than 100 characters; no trimming performed.")
-			}
-		} else {
-			log.Fatalf("Field 'code' not found or is not a string in registry bytecode: %s", key)
-		}
-	} else {
-		log.Fatalf("Key %s not found in registry bytecode", key)
-	}
-
-	expectedData, err = json.Marshal(weth9ExpectedData)
+	data, err = json.Marshal(weth9Data)
 	if err != nil {
 		fmt.Printf("error marshalling regenerated bytecode: %s", err)
```

Comment:
Here we use "regenerated" even though the function is generic. 

Might be nice to return an error form this function, and then the caller has the context to print out information about which data was being manipulated.

Reply:
Done